dspace/config/crosswalks/oai/metadataFormats/oai_openaire.xsl

```diff
@@ -0,0 +1,1484 @@
+
+
+    <!-- dc:description -->
+    <!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_description.html -->
+    <xsl:template
+        match="doc:element[@name='dc']/doc:element[@name='description']/doc:element[@name='abstract']/doc:element"
+        mode="dc">
+        <dc:description>
+            <xsl:call-template name="xmlLanguage">
+                <xsl:with-param name="name" select="@name"/>
+            </xsl:call-template>
+            <xsl:value-of select="./doc:field[@name='value']"/>
+        </dc:description>
+    </xsl:template>
+
+    <!-- dc:format -->
+    <!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_size.html -->
+    <xsl:template match="doc:element[@name='bundles']/doc:element[@name='bundle']" mode="dc">
+        <xsl:if test="doc:field[@name='name' and text()='ORIGINAL']">
+            <xsl:for-each select="doc:element[@name='bitstreams']/doc:element[@name='bitstream']">
+                <xsl:apply-templates select="." mode="dc"/>
+            </xsl:for-each>
+        </xsl:if>
+    </xsl:template>
+    <xsl:template match="doc:element[@name='bitstreams']/doc:element[@name='bitstream']" mode="dc">
+        <dc:format>
+            <xsl:value-of select="doc:field[@name='format']"/>
+        </dc:format>
+    </xsl:template>
+
+
+    <!-- datacite:sizes -->
+    <!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_size.html -->
+    <xsl:template match="doc:element[@name='bundles']/doc:element[@name='bundle']" mode="datacite">
+        <datacite:sizes>
+            <xsl:if test="doc:field[@name='name' and text()='ORIGINAL']">
+                <xsl:for-each select="doc:element[@name='bitstreams']/doc:element[@name='bitstream']">
+                    <xsl:apply-templates select="." mode="datacite"/>
+                </xsl:for-each>
+            </xsl:if>
+        </datacite:sizes>
+    </xsl:template>
+    
+     <!-- datacite:size -->
+    <xsl:template match="doc:element[@name='bitstreams']/doc:element[@name='bitstream']" mode="datacite">
+        <datacite:size>
+            <xsl:value-of select="concat(doc:field[@name='size'],' bytes')"/>
+        </datacite:size>
+    </xsl:template>
+
+   <!-- oaire:file -->
+   <!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_filelocation.html -->
+    <xsl:template match="doc:element[@name='bundles']/doc:element[@name='bundle']" mode="oaire">
+       <!-- only consider ORIGINAL bundle -->
+        <xsl:if test="doc:field[@name='name' and text()='ORIGINAL']">
+            <xsl:for-each select="doc:element[@name='bitstreams']/doc:element[@name='bitstream']">
+                <xsl:apply-templates select="." mode="oaire"/>
+            </xsl:for-each>
+        </xsl:if>
+    </xsl:template>
+
+    <xsl:template match="doc:element[@name='bitstreams']/doc:element[@name='bitstream']" mode="oaire">
```

Comment:
Please describe this template with a comment.

Reply:
Just add one description.
https://github.com/DSpace/DSpace/pull/2592/commits/7814a9a1df632f03d9d97191637e722078bba317